deploy/roles/elasticsearch/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
-es_version: 2.3
+es_version: 2.x
 es_full_version: 2.3.4
 es_data_directory: /opt/elasticsearch/data
 es_transport_ports: "9300:9400"
 es_http_ports: "9200:9299"
+proxy_server_deployed: true
```

Comment:
@stirlingv See below, this variable can probably be removed, but if we did want to keep it it should live in group_vars/all/main.yml, since it is a "global" variable not just related to Elasticsearch but rather the state of the Enclave as a whole.